### ISSUE TITLE
Cron ignores cron expression schedule

### ISSUE THREAD

**Issue by meticoeus** [serverless-stack/serverless-stack#1259]
I am creating a cron job lambda function with a non-trivial schedule. When I deploy the function it creates the stack successfully. However it is running every minute instead of following the schedule. When I reviewed the CloudFormation stack's created resources the cron schedule for the event rule was '* * * * ? *' instead of the schedule I defined in code.

I was able to workaround for now by manually editing the the cron expression to '0/30 11-2 * * ? *' in the AWS console which showed the expected times of execution and saved successfully. This looks like a bug in either sst or possibly CDK.

Here is the stack code:

```ts
// ./stack/CronStack.ts:
import * as cdk from '@aws-cdk/core'
import * as sst from '@serverless-stack/resources'

export class CronStack extends sst.Stack {
  constructor(scope: sst.App, id: string, props?: sst.StackProps) {
    super(scope, id, props)
    new sst.Cron(this, 'ScheduleSomeFunction', {

// This gets turned into 'cron(* * * * ? *)' in deployed event rule:
      schedule: '0/30 11-2 * * ? *',

      job: new sst.Function(stack, 'SomeFunction', {
        handler: 'src/someFunction.handler',
        timeout: cdk.Duration.seconds(29),
      }),
    })
  }
}

// ./stack/index.ts
import { CronStack } from './CronStack'
import * as sst from '@serverless-stack/resources'

export default function main(app: sst.App): void {
  app.setDefaultFunctionProps({
    runtime: 'nodejs14.x',
  })

  new CronStack(app, 'assistant-stack')
}
```

**Comment by Manitej66** [comment]
Have you tried with AWS CDK V2?

**Comment by meticoeus** [comment]
> Have you tried with AWS CDK V2?

If you mean if I've tried using CDK directly rather than SST for this project, then no. I created a new project with SST last week then deployed it. My project is a pair of simple cron job lambdas that execute code I was previously running locally.

I partially posted this to inform anyone else who runs across this of the issue and the workaround in case others are using SST for non-trivial cron jobs. My deployment worked and my functions were running with no obvious problems; only the non-obvious problem of running every minute instead of when I needed them to, which would have caused much higher charges against APIs those lambdas call than I'd be expecting.
